packages/frontend/src/components/wallet/Wallet.js

```diff
@@ -39,6 +39,14 @@ const StyledContainer = styled(Container)`
             margin-top: -15px;
         }
     }
+    
+    .coingecko {
+        color: #B4B4B4;
+        line-height: 150%;
```

Comment:
why the `line-height: 150%;`?

Reply:
We can skip the line-height if we apply the [margin](https://github.com/near/near-wallet/pull/2650/files#r870748785)